Gemfile

```diff
@@ -59,4 +61,7 @@ group :test do
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+
+# Rubocop
+gem 'rubocop', require: false
```

Comment:
I prefer to tag gems using specifier `'~> mayor.minor.revision'
```rb
gem 'rubocop', '~> 0.66.0', require: false
```
Tag gem version it's seems to me a good practice, to avoid automatic update gems. If you like, you can read [this article](https://everydayrails.com/2012/09/11/bundler-rails-specify-versions.html)